src/assets/i18n/en.json5

```diff
@@ -2652,16 +2674,54 @@
 
   "item.version.create.modal.button.confirm.tooltip": "Create new version",
 
+  "item.qa.withdrawn-reinstate.modal.button.confirm.tooltip": "Send request",
+
+  "qa-withdrown.create.modal.button.confirm": "Withdraw",
+
+  "qa-reinstate.create.modal.button.confirm": "Reinstate",
+
   "item.version.create.modal.button.cancel": "Cancel",
 
+  "item.qa.withdrawn-reinstate.create.modal.button.cancel": "Cancel",
+
   "item.version.create.modal.button.cancel.tooltip": "Do not create new version",
 
+  "item.qa.withdrawn-reinstate.create.modal.button.cancel.tooltip": "Do not send request",
+
   "item.version.create.modal.form.summary.label": "Summary",
 
+  "qa-withdrawn.create.modal.form.summary.label": "You are requesting to withdraw this item",
+
+  "qa-withdrawn.create.modal.form.summary2.label": "Please enter the reason for the withdrawal",
+
+  "qa-reinstate.create.modal.form.summary.label": "You are requesting to reinstate this item",
+
+  "qa-reinstate.create.modal.form.summary2.label": "Please enter the reason for the reinstatment",
+
   "item.version.create.modal.form.summary.placeholder": "Insert the summary for the new version",
 
+  "qa-withdrown.modal.form.summary.placeholder": "Enter the reason for the withdrawal",
+
+  "qa-reinstate.modal.form.summary.placeholder": "Enter the reason for the reinstate",
+
   "item.version.create.modal.submitted.header": "Creating new version...",
 
+  "item.qa.withdrawn.modal.submitted.header": "Sending withdrawn request...",
+
+  "correction-type.manage-relation.action.notification.reinstate": "Reinstate request sent.",
+
+  "correction-type.manage-relation.action.notification.withdrawn": "Withdraw request sent.",
+
+  "qa-event-notification.check.notification-withdrawn.user": "There are {{num}} items pending review(s) to check.",
+
+  "qa-event-notification.check.notification-withdrawn.admin": "There are {{num}} {{source}} feedback(s) pending.",
+
+  "qa-event-notification.check.notification-reinstate": "There are {{num}} pending reinstatement(s) pending for this item.",
```

Comment:
This should be reworded to say:  "There are {{num}} pending reinstatement request(s) for this item."